viewer/views/connections.jade

```diff
@@ -352,10 +353,7 @@ block content
             focus_node = null,
             highlight_node = null;
 
-        node.on("mouseover", function(d) {
-          set_highlight(d);
-        })
-        .on("mousedown", function(d) {
+        node.on("mousedown", function(d) {
```

Comment:
Hmm, this shouldn't need to change

node.on().on()  should work vs
node.on(); node.on();